packages/protocol/test/governance/governance.ts

```diff
@@ -1511,10 +1779,11 @@ contract('Governance', (accounts: string[]) => {
               // @ts-ignore: TODO(mcortesi) fix typings for TransactionDetails
               { value: minDeposit }
             )
+            await mockBondedDeposits.setWeight(account, weight)
```

Comment:
Can these be pulled into a `beforeEach` in `when the proposal cannot execute successfully` to dedupe?